fw/executor/main.go

```diff
@@ -70,7 +70,7 @@ func Main(args []string) {
 	sigChannel := make(chan os.Signal, 1)
 	signal.Notify(sigChannel, os.Interrupt, syscall.SIGTERM)
 	receivedSig := <-sigChannel
-	core.LogInfo("Main", "Received signal ", receivedSig, " - exiting")
+	core.LogInfo(yanfd, "Received signal ", receivedSig, " - exiting")
```

Comment:
Do we still call it `yanfd`?

Reply:
Since we have a paper, I'm inclined to keep the name ...